src/webextension/widgets/filter-input.js

```diff
@@ -48,20 +47,17 @@ export default class FilterInput extends React.Component {
     const {className, onChange, value, disabled, ...props} = this.props;
     const disabledClass = disabled ? ` ${styles.disabled}` : "";
     const finalClassName = (
-      `${styles.inputWrapper}${disabledClass} ${className}`
+      `${styles.filter} ${styles.inputWrapper}${disabledClass} ${className}`
     ).trimRight();
 
     return (
       <div className={finalClassName}>
         <input type="search" {...props} disabled={disabled}
                value={this.state.value}
                onChange={(e) => this.updateValue(e.target.value)}/>
-        <Localized id="filter-input-clear">
-          <button type="button" disabled={disabled}
-                  onClick={() => this.updateValue("")}>
-            cLEAr
-          </button>
-        </Localized>
+        <button type="button" title="Clear" disabled={disabled}
```

Comment:
`"Clear"` here should be changed to a fake-localization (i.e. `"cLEAr"`) and then the `<Localized>` wrapper should be restored. Then you just need to adjust the localization in `widgets.ftl`.

Reply:
Localization restored 👌